scripts/setup-toolchain.sh

```
@@ -0,0 +1,184 @@
+for arg in "$@"; do
+    case "$arg" in
+        --check) MODE="check" ;;
+        -h|--help)
+            sed -n '1,30p' "$0" | sed 's/^#//'
+            exit 0
+            ;;
+        *) echo "unknown flag: $arg" >&2; exit 2 ;;
+    esac
+done
+
+PROTOC_EXPECTED_MAJOR="25"
```

Shellcheck (0.11.0)

[warning] PROTOC_EXPECTED_MAJOR appears unused. Verify use (or export if used externally).

(SC2034)